pages/_app.tsx

```diff
@@ -0,0 +1,28 @@
+import { ChakraProvider } from "@chakra-ui/react";
+import type { AppProps } from "next/app";
+import { ToastContainer } from "react-toastify";
+import Navbar from "../components/Navbar";
+import "react-toastify/dist/ReactToastify.css";
+
+import { theme } from "../styles/theme";
+
+function MyApp({ Component, pageProps }: AppProps) {
+  return (
+    <ChakraProvider theme={theme}>
+      <Navbar>
+        <Component {...pageProps} />
+      </Navbar>
+      <ToastContainer
+        position="top-right"
+        hideProgressBar={false}
+        autoClose={3000}
+        newestOnTop={true}
```

Comment:
No JSX se você declara uma prop, por padrão ela vai assumir um valor verdadeiro. Você precisa apenas especificar seu valor, caso ele for falso.

```suggestion
        hideProgressBar={false}
        autoClose={3000}
        newestOnTop
```